doc/ref_cert/RC2/chapters/chapter03.md

```diff
@@ -33,13 +32,13 @@ the network settings:
 
 To deploy your own CI toolchain running CNTT Compliance:
 ```bash
-virtualenv functest-kubernetes
+virtualenv functest-kubernetes --system-site-packages
 . functest-kubernetes/bin/activate
 pip install ansible
 ansible-galaxy install collivier.xtesting
 ansible-galaxy collection install ansible.posix community.general community.grafana community.kubernetes community.docker community.postgresql
 git clone https://gerrit.opnfv.org/gerrit/functest-kubernetes functest-kubernetes-src
-(cd functest-kubernetes-src && git checkout -b stable/leguer origin/stable/leguer)
+(cd functest-kubernetes-src && git checkout -b stable/v1.21 origin/stable/v1.21)
```

Comment:
Is it possible to refer to the version number specified in RA2 READMe.md?

Reply:
yes wil do in a separated change (I shouldn't have updated it here in the best opensource rules)